projects/packages/my-jetpack/_inc/components/welcome-flow/index.tsx

```diff
@@ -56,36 +56,20 @@ const WelcomeFlow: FC = () => {
 	}, [ recordEvent, currentStep, isUserConnected, isSiteConnected, dismissWelcomeBanner ] );
 
 	const handleEvaluation = useCallback(
-		( values: { [ key in EvaluationAreas ]: boolean } ) => {
-			const siteGoals = Object.keys( values ).filter( key => values[ key ] );
+		async ( values: { [ key in EvaluationAreas ]: boolean } ) => {
+			const goals = Object.keys( values ).filter( key => values[ key ] );
 
 			setIsProcessingEvaluation( true );
-			recordEvent( 'jetpack_myjetpack_welcome_banner_submit_evaluation', { siteGoals } );
-			submitEvaluation(
-				{ queryParams: { goals: siteGoals } },
-				{
-					onSuccess: recommendations => {
-						// Convert object to array of [key, value] pairs
-						const recommendedModules = Object.entries( recommendations )
-							.sort( ( a, b ) => b[ 1 ] - a[ 1 ] )
-							.map( entry => entry[ 0 ] );
+			recordEvent( 'jetpack_myjetpack_welcome_banner_submit_evaluation', { goals } );
```

Comment:
For Tracks events, it's a convention to have the verb at the end of the event name in a non conjugated form

For reference: https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-analytics/README.md#naming-conventions

```suggestion
			recordEvent( 'jetpack_myjetpack_welcome_banner_evaluation_submit', { goals } );
```

Reply:
Great, thanks for the reference especially, will remember that - I wish linter could pick that up